android/sentry.properties

```diff
@@ -1,4 +1,4 @@
 defaults.url=https://sentry.io/
 defaults.org=testorg-az
-defaults.project=mobile-webinar
-auth.token=XXXX
+defaults.project=reactnativeadamtest
+auth.token=bbece28fc4b847baaed54aa3518c5e7ced9bbaaac4404fc48f81dea11347caae
```

Comment:
@sentryadam0000345 remove this Auth Token from source code. Then delete it from your Settings in Sentry.io and generate a new one and start using that.

Reply:
Done